Lykke.Job.LykkeJob/src/Lykke.Job.LykkeJob.Services/ShutdownManager.cs

```diff
@@ -14,23 +15,27 @@ namespace Lykke.Job.LykkeJob.Services
     public class ShutdownManager : IShutdownManager
     {
         private readonly ILog _log;
-        private readonly List<IStopable> _items = new List<IStopable>();
+        private readonly IEnumerable<IStopable> _items;
 
-        public ShutdownManager(ILog log)
+        public ShutdownManager(ILog log, IEnumerable<IStopable> items)
```

Comment:
Is this IEnumerable would be generated by Autofaq automatically?
If yes, then it's ok, if no - then we force calling code to know the whole list of stopppable items. In previous implementation all such items could register themselves in this manager - without explicit list of items to be stopped.

Reply:
Autofac collects all IStopable instances and sends to the ctor. http://autofaccn.readthedocs.io/en/latest/resolve/relationships.html#enumeration-ienumerable-b-ilist-b-icollection-b

Reply:
Ok. Cool.